### PR TITLE
Make multigrid default and add settings disabling multigrid specific behavior

### PR DESCRIPTION
**Pull request by Kethku**
## What kind of change does this PR introduce?
- Fix

## Did this PR introduce a breaking change? 
- Yes, multigrid is now default behavior

### CODE REVIEW

File: src/renderer/mod.rs [hunk 1]
```diff
@@ -30,7 +30,9 @@ pub use rendered_window::{LineFragment, RenderedWindow, WindowDrawCommand, Windo
 #[derive(SettingGroup, Clone)]
 pub struct RendererSettings {
     position_animation_length: f32,
+    window_animation: bool,
     scroll_animation_length: f32,
+    smooth_scrolling: bool,
```
Comment (fredizzimo) [inline src/renderer/mod.rs:35]: It seems like only the `cmd_line.rs` file deals with multigrid stuff. The rest adds `window_animation` and `smoth_scrolling`, but no corresponding commandline options or environment variables. Is this intentional?

File: src/cmd_line.rs [hunk 2]
```diff
@@ -109,9 +109,9 @@ pub fn handle_command_line_arguments(args: Vec<String>) -> Result<(), String> {
                 .help("Maximize the window"),
         )
         .arg(
-            Arg::with_name("multi_grid")
-                .long("multigrid")
-                .help("Enable Multigrid"),
+            Arg::with_name("no_multi_grid")
+                .long("nomultigrid")
```
Comment (last-partizan) [inline src/cmd_line.rs:113]:
```suggestion
                .long("no-multigrid")
```

Maybe this is better choice

File: src/renderer/mod.rs (hunk 1 above)
Comment (Kethku) [reply src/renderer/mod.rs:35]: No this makes sense. When multigrid is disabled, the rendering system renders into a single window and doesn't send relevant scrolling events. So no cmd_line argument is needed.

File: src/cmd_line.rs (hunk 2 above)
Comment (Kethku) [reply src/cmd_line.rs:113]: I debated here, but there were other existing settings which concatted noidle so I went with this. Instead I think I'll add a dash to the other items

Comment (Kethku) [reply src/cmd_line.rs]: Done

**Comment by last-partizan** [comment]
Is there something else needs to be done before merging this?

**Comment by bmulholland** [comment]
If this were merged, I'd have to disable it immediately because of https://github.com/neovide/neovide/issues/1503

**Comment by falcucci** [comment]
we still have visual issues related to the `--multigrid` functionality already reported at #1819 #1508 

we need to analise about what is going in the MsgArea behavior when multigrid is enabled.

**Comment by Kethku** [comment]
Closing in favor of improve-render-loop branch

**Pull request closed without merging**